include/boost/histogram/python/accumulators_ostream.hpp

```diff
@@ -8,6 +8,10 @@
 
```

Comment:
in boost.histogram the  ostream operators must be in a separate header, but here you could just move the code to the corresponding accumulator. it is more obvious this way.

Reply:
Ours are customized anyway, so we can. Though just moving it to `python/accumulators/ostream.hpp` would help (and sum's ostream is in there too).

Reply:
I moved it, and it seems like it's reasonably ordered, up to you if you want it moved further.